run.sh

```diff
@@ -64,10 +104,14 @@ case "$deployment_action" in
     "deploy_osh")
         deploy_osh
         ;;
-    "add_compute")
+    "add_openstacK_compute")
```

Comment:
Typo capitalisation.

Reply:
for option it should be small case.

Reply:
`openstacK` -> `openstack`

Notice the last letter

Reply:
Yes done